src/components/Votes/ApprovalProposalVotesList/ApprovalProposalSingleVote.tsx

```diff
@@ -81,8 +94,24 @@ export default function ApprovalProposalSingleVote({ vote }: { vote: Vote }) {
                 </>
               )}
             </div>
-            <div className={"font-semibold text-secondary"}>
-              <TokenAmountDisplay amount={weight} />
+            <div className={"font-semibold text-primary"}>
```

Comment:
Maybe something to get design help on but two things here:

1. I don't think the rightmost token amount looks like it's hoverable or that it has a tooltip to show. The only reason I knew to hover was because I'm reviewing this code. Maybe we need another design pass to make the hover more obvious.
2. I don't like how hovering the tooltip however the voter card at the same time. We should get rid of the voter card entirely, there was a conversation around this earlier last week. Instead, we would make the voter hoverable and link out to the delegate page. cc @andreitr on that one

<img width="869" alt="Screenshot 2025-01-28 at 9 50 27 AM" src="https://github.com/user-attachments/assets/49166d09-25a4-4f98-9d7e-711988f5030f" />